backtracking/sudoku_solve.cpp

```diff
@@ -25,18 +23,19 @@ bool isPossible(int mat[][9], int i, int j, int no) {
 
     return true;
 }
+
 void printMat(int mat[][9]) {

```

Comment:
1. Function documentation missing
2. Arrays to be passed as pointers

Reply:
> 2\. Arrays to be passed as pointers

Do you mean like:

1\.

    int **mat

2\.

    const int **mat

3\.

    const int *mat
Which one is correct? Or is there something better?

Reply:
you can use any of the above.
1 -> if `mat` values are being modified by the function
2 -> if `mat` values are not being modified by the function 
3 -> this is a pointer to 1D array => Any N-D matrix can be stored and accessed from a 1D array -> This is fine provided it is used consistently throughout the code and you know how to implement it. Good opportunity for you to learn the approach or if not, refrain from this.